charts/cloudzero-agent/README.md

```diff
@@ -17,255 +17,36 @@ For the latest release, see [Releases](https://github.com/Cloudzero/cloudzero-ch
 
 ## Installation
 
-### Adding Helm Repository Information
+### Easy Install (Most teams will use this!)
 
 To use the chart or a beta version, you must add the repository to Helm. Refer to the [`helm repo`](https://helm.sh/docs/helm/helm_repo/) documentation for command details.
 
-#### 1. Add the Helm Chart Repository
```

Comment:
Looking to reduce complexity here. Here are two commands to get the agent spun with some sensible defaults.

Reply:
I'm all in favor of having an "easy install" option. but, I feel that one of our issues so far is that "sensible defaults" varies a lot. for example, in previous versions, no labels were enabled by default. so customers would install, and then complain later their labels were missing, and we found that users weren't digging into the docs too much to see that labels had to be manually enabled.

in the recent beta, our solution to this was to force users to make choices at install time, so that a user needs to specify if they want labels or not, which label they want, etc. so, installation might be a bit harder, but the user would be forced to make the (hopefully) correct choices.

but all of that is to say, I am all for trying to focus in on a sensible default setup